analysis/topEFT/datacard_maker.py

```diff
@@ -299,7 +300,7 @@ def getHist(d_hists,name):
             xmax = h.GetXaxis().GetXmax()
             xwidth = h.GetXaxis().GetBinWidth(1)
             h.GetXaxis().SetRangeUser(xmin, xmax + xwidth) #Include overflow bin in ROOT
-            return h
+            return deepcopy(h) # to protect d_hists from modifications 
```

Comment:
Interesting. Was this a bug before, or are these improvements the only thing that is sensitive to modifications?

Reply:
I *think* only the new improvements are affected. The only problem should show up when one modifies the histogram that is produced, like we do with the `quad` terms. Since I do it once once to see if the WCs have an effect and then another one to actually store the histos, I think now it's necessary to have this